src/renderer/components/reader/ReaderMenu.tsx

```diff
@@ -39,23 +40,41 @@ interface Props extends TranslatorProps {
 interface State {
     openedSection: number;
     bookmarkToUpdate: number;
+    pageError: boolean;
+    refreshError: boolean;
 }
 
 export class ReaderMenu extends React.Component<Props, State> {
+    private goToRef: any;
     public constructor(props: Props) {
         super(props);
 
         this.state = {
             openedSection: undefined,
             bookmarkToUpdate: undefined,
+            pageError: false,
+            refreshError: false,
         };
 
         this.closeBookarkEditForm = this.closeBookarkEditForm.bind(this);
+        this.handleSubmitPage = this.handleSubmitPage.bind(this);
+    }
+
+    public componentDidUpdate() {
+        if (this.state.refreshError) {
```

Comment:
The error message is too "sticky", it should disappear as soon as the user starts typing a new value in the input text field for "goto page".

![ThoriumGotoPageStickyError](https://user-images.githubusercontent.com/625201/64261567-a2495d00-cf24-11e9-9c20-eb67e81a8ab0.png)

Reply:
Fixed via https://github.com/readium/readium-desktop/commit/d38073a801fbf57a96525287a026e9b0da06f196